build/repo.targets

```diff
@@ -0,0 +1,39 @@
+<Project>
+
+  <PropertyGroup>
+      <SiteExtensionProjectDirectory>$(RepositoryRoot)src\Microsoft.AspNetCore.AzureAppServices.TestBundle\</SiteExtensionProjectDirectory>
+      <PublishFeed Condition="$(PublishFeed) == ''">https://dotnet.myget.org/F/aspnetcore-ci-dev/</PublishFeed>
+      <DotnetChannel>master</DotnetChannel>
+      <DotnetVersion>coherent</DotnetVersion>
+  </PropertyGroup>
+
+  <Target Name="BuildSiteExtension">
+
+    <PropertyGroup>
+      <SiteExtensionWorkingDirectory>$(RepositoryRoot).dotnet\</SiteExtensionWorkingDirectory>
+      <CliVersionRelativePath>build\dotnet.version</CliVersionRelativePath>
+    </PropertyGroup>
+
+    <RemoveDir Directories="$(SiteExtensionWorkingDirectory)" />
+    <Exec Command="$(MSBuildThisFileDirectory)\dotnet-install.cmd -Channel $(DotnetChannel) -Version $(DotnetVersion) -Architecture x86 -InstallDir $(SiteExtensionWorkingDirectory) -NoPath" />
+    <MSBuild Projects="$(SiteExtensionProjectDirectory)Microsoft.AspNetCore.AzureAppServices.TestBundle.csproj"
+      Targets="Restore;Pack"
+      Properties="DotnetHomeDirectory=$(SiteExtensionWorkingDirectory)" />
+
+    <Exec Command="$(SiteExtensionWorkingDirectory)dotnet --version > $(RepositoryRoot)artifacts\$(CliVersionRelativePath)" />
```

Comment:
What's this currently used for? It doesn't seem to be used anywhere else in this PR.